src/components/Countries.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const Countries = ({ countries, affected }) => (
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name, kindly implement this feature so your app can be searchable very easily

Reply:
Hi @Ismailco, it seems I do not understand this... Kindly elaborate

Reply:
@Ibnballo1, In the home page you should include functionality that filters the categories (countries in your case) by a given parameter for example the name of the country, like adding a search input that filters the listed countries and only shows the countries with that name (searched name)

![Screenshot_20221008_194538](https://user-images.githubusercontent.com/25227022/194722955-a0cf7dc6-3845-4417-878e-1a0054894977.png)